audio-worklet/basic/js/noise-generator.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+registerProcessor('noise-generator', class Noise extends AudioWorkletProcessor {
+  static get parameterDescriptors () {
+    return [{
+      name: 'gain',
+      defaultValue: 0.25,
+      minValue: 0,
+      maxValue: 1
+    }];
+  }
+
+  constructor () {
+    super();
+  }
+
+  process (input, output, parameters) {
+    let outputChannelData = output.getChannelData(0);
+    let gain = parameters.gain;
+    for (let i = 0; i < 128; ++i) {
```

Comment:
Should you replace 128 with the length of `outputChannelData`, even though we know it's 128?